test/unit/node/update.test.ts

```diff
@@ -45,9 +45,11 @@ describe("update", () => {
       return response.end()
     }
 
-    // NOTES subpath match /redirect/${number}
-    // if number === 0, resolve it
-    if (request.url.match(/\/redirect\/[0-9]/)) {
+    // Checks if url matches /redirect/${number}
+    // with optional trailing slash
+    let redirectSlashNumberRegExp = new RegExp(/\/redirect\/([0-9]+)(\/)?$/)
+
+    if (request.url.match(redirectSlashNumberRegExp)) {
```

Comment:
If you want to use the match group you captured you would need:

```
const match = request.url.match(redirectSlashNumberRegExp)
if (match) {
  const currentRedirectNumber = parseInt(match[1])
  // rest of the code
}
```

This could replace the `split` logic.  `match` is an object with the total match in [0], the first capture group in [1], second in [2], etc.

Reply:
Ooo this is a wonderful idea! I'll do this